src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -375,6 +376,12 @@ PeerImp::json()
         }
     }
 
+    ret[jss::metrics] = Json::Value(Json::objectValue);
+    ret[jss::metrics][jss::total_bytes_recv] = Json::UInt(metrics_.recv.total_bytes());
+    ret[jss::metrics][jss::total_bytes_sent] = Json::UInt(metrics_.sent.total_bytes());
```

Comment:
I did notice that `Json::UInt` is a 32-bit int, wondering if there's a better type here given that `total_bytes` is a 64-bit int.